app/furniture/marketplace/cart_product.rb

```diff
@@ -8,13 +8,23 @@ class CartProduct < Record
     delegate :shopper, to: :cart
 
     belongs_to :product, inverse_of: :cart_products
-    delegate :name, :description, :marketplace, :price, :price_cents, to: :product
+    delegate :name, :description, :marketplace, :price, :price_cents, :tax_rates, to: :product
 
     validates_uniqueness_of :product, scope: :cart_id
     validate :editable_cart
 
     attribute :quantity, :integer, default: 0
 
+    def tax_amount
+      price_total * (tax_rates.sum(0) do |tax_rate|
+        tax_rate.tax_rate
```

Comment:
Same as above, I think this can be `price_total * (tax_rates.sum(&:tax_rate) / 100)`